compose/web/benchmark/build.gradle

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright 2021 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import androidx.build.AndroidXUiPlugin
+import androidx.build.LibraryGroups
+import androidx.build.Publish
+
+buildscript {
+    repositories {
+        gradlePluginPortal()
+    }
+
+    dependencies {
+        classpath "com.github.node-gradle:gradle-node-plugin:3.0.1"
+    }
+}
+
+plugins {
+    id("AndroidXPlugin")
+    id("AndroidXUiPlugin")
+    id("kotlin-multiplatform")
+}
+
+apply plugin: 'com.github.node-gradle.node'
+
+AndroidXUiPlugin.applyAndConfigureKotlinPlugin(project)
+dependencies {
+    kotlinPlugin project(":compose:compiler:compiler")
+}
+
+node {
+    download = true
+}
+
+kotlin {
+    jvm()
+    js(IR) {
+        browser() {
+            testTask {
+                useKarma {
+                    useChromeHeadless()
+                }
+            }
+        }
+        binaries.executable()
+    }
+
+    sourceSets {
+        commonMain {
+            dependencies {
+                implementation project(":compose:runtime:runtime")
+                implementation project(":compose:web")
+            }
+        }
+
+        jsMain {
+            dependencies {
+                implementation(kotlin("stdlib-js"))
+            }
+        }
+
+        jsTest {
+            dependencies {
+                implementation kotlin("test-js")
+            }
+        }
+    }
+}
+
+androidx {
+    name = "Compose Web Benchmar"
```

Comment:
Benchmark

Reply:
I removed this 'androidx' section since we don't need it.